### ISSUE TITLE
Get latest information under each category, create markdown files for each category

### ISSUE THREAD

**Issue by arsulegai** [hyperledger-tooling/start-here-hyperledger#148]
Referring to the idea proposed here in the wiki
https://wiki.hyperledger.org/display/~davidwboswell/Main+page+edits

The tool shall fetch latest PRs, latest issues, latest contributors etc information. Place them in markdown files. That can be fetched directly into the wiki space.

**Comment by arsulegai** [comment]
Work is partially done.
TODO:
1. Update the output from the `hyperledger-updates` repository if the templates in `start-here-hyperledger` expects additional fields.
2. Add an option to pull recent contributors in `hyperledger-updates` and enable polling in `start-here-hyperledger`.

**Comment by abhinandanudupa** [comment]
Can you please assign to me this issue?

**Comment by arsulegai** [comment]
@abhinandanudupa please feel free to raise PRs for the work items. You could add a comment here to let others know which part of the issue you are working on.

**Comment by abhinandanudupa** [comment]
Alright, I will start working soon and update the details here.

**Comment by abhinandanudupa** [comment]
Just to confirm - `hyperledger-updates` is `github-updates` repo as linked [here](https://start-here.hyperledger.org/) right?

**Comment by abhinandanudupa** [comment]
To solve 
> 2. Add an option to pull recent contributors in `hyperledger-updates` and enable polling in `start-here-hyperledger`.

I will have to write a function like [this](https://github.com/hyperledger-tooling/github-updates/blob/main/internal/pkg/client/ghclient.go#L91), right?

And then configure the `config.yaml` like [this](https://github.com/hyperledger-tooling/github-updates/blob/main/assets/config.yaml#L31)?

**Comment by arsulegai** [comment]
@abhinandanudupa that's right. The expectation for the trending/recent contributors is to produce information in this folder https://github.com/hyperledger-tooling/start-here-hyperledger/tree/main/trending. These files can be directly populated into the wiki and the wiki is automatically kept up-to-date.

**Comment by abhinandanudupa** [comment]
Ok, then I will start working on it.

**Comment by abhinandanudupa** [comment]
I am able to generate the contributor list but I am not sure on what should be displayed for each contributor. Can you tell me what is necessary to be displayed?

And can you tell me how the data in `github-updates` repository (that I have modified to get contributors data along with the rest)  will be updated in this repository?

**Comment by abhinandanudupa** [comment]
@arsulegai Here is the [PR](https://github.com/hyperledger-tooling/github-updates/pull/49) that would solve this, check it out
